src/common/index.ts

```diff
@@ -11,8 +11,11 @@
  */
 
 export const devworkspaceVersion = 'v1alpha2';
-export const group = 'workspace.devfile.io';
+export const devworkspaceGroup = 'workspace.devfile.io';
 export const devworkspaceSubresource = 'devworkspaces';
+export const devworkspaceTemplateSubresource = 'devworkspacetemplates';
+export const projectsId = 'projects';
+export const projectRequestId = 'projectrequests';
 
 export const openshiftIdentifier = 'project.openshift.io';
 export const devworkspaceIdentifier = 'workspace.devfile.io';
```

Comment:
let's name it devWorkspaceApiGroup

Reply:
Well, what is the difference between this and `devworkspaceGroup'?
`

Reply:
No difference, I just didn't realize I had defined it :laughing: